requirements.txt

```diff
@@ -1,2 +1,4 @@
 pytz
 requests
+Naked
+shutil
```

Comment:
Error
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://pypi.org/simple/shutil/

Reply:
It seemed we don't need it.